terraform/kubernetes-cluster/variables.tf

```diff
@@ -0,0 +1,4 @@
+variable "delta_cert_expiration_seconds" {
+  description = "Number of seconds until the Kubernetes certificate expires"
+  default     = 31536000 # 1 year
```

Comment:
Better to set at least to 10 years

Reply:
It should be address at: https://github.com/franciscoprin/kubernetes-the-hard-way-aws-terraform/pull/1/commits/6043fd40174d75c37aa48c1d0ba3d86c5eebf41f

Reply:
Could I ask why 10 years is better? Isn't it insecure to have these files rotated regularly?